.circleci/config.yml

```diff
@@ -26,7 +26,7 @@ jobs:
             go get sigs.k8s.io/kind
             cp /go/bin/kind /binaries/kind
       - save_cache:
-          key: v1-binaries
+          key: v2-binaries
```

Comment:
is this change directly pertinent to the Pull Request of adding ro-cert-manager to incubator? Or is it included to not have yet another PR for a small change? (Trying to see if this should be under the microscope or is correlated to the files in the rest of the PR)

Reply:
I discovered during testing that a new version of kind was released since the last build, and this is intended to force a rebuild of the kind binary.  Circle doesn't invalidate that cache very often (which in this case is just the kind binary).

So it is just avoiding an additional PR :-D